src/Keter/Config/V10.hs

```diff
@@ -182,6 +185,7 @@ instance ParseYamlFile KeterConfig where
             <*> o .:? "missing-host-response-file"
             <*> o .:? "unknown-host-response-file"
             <*> o .:? "proxy-exception-response-file"
+            <*> o .:? "rotate-logs" .!= True
```

Comment:
I think we should update /etc/keter-config.yaml to reflect the existence of this option. Maybe we set it to false there by default as well as emmitting to stderr is more standard unix behavior, but unset we still want to set it to true so existing users don't lose logs

Reply:
Added